uui/components/fileUpload/FileCard.tsx

```diff
@@ -13,12 +13,25 @@ import { ReactComponent as RemoveIcon } from '@epam/assets/icons/common/navigati
 import { ReactComponent as ErrorIcon } from '../../icons/notification-error-fill-10.svg';
 
 export interface FileCardItem extends Partial<File & FileUploadResponse> {
+    /*
+    * Defines upload progress.
```

Comment:
Also add - 'It should be number value in range from 0 to 100', where 100 means that file fully loaded.